backend/data/static/datasets_clean.py

```diff
@@ -0,0 +1,222 @@
+    "res": "LSOA",
+    "key_col": "Area Codes",
+    "key_is_code": True,
+    "keep_cols": [],
+    "csv_name": "over_65_count",
+}
+
+la_over_65_dict = {
+    "data": raw.OVER_65_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 3",
+    "key_is_code": False,
+    "rename_dict": {"Unnamed: 14": "over_65_count"},
+    "keep_cols": [],
+    "csv_name": "over_65_count",
+}
+
+lsoa_imd_dict = {
+    "data": raw.IMD_LSOA,
+    "res": "LSOA",
+    "key_col": "lsoa11cd",
+    "key_is_code": True,
+    "keep_cols": ["LSOA11CD", "wimd_2019"],
+    "csv_name": "wimd_2019",
+}
+
+la_imd_dict = {
+    "data": raw.IMD_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 0",
+    "key_is_code": False,
+    "keep_cols": [],
+    "csv_name": "wimd_2019",
+}
+
+lsoa_popdensity_dict = {
+    "data": raw.POPDENSITY_LSOA,
+    "res": "LSOA",
+    "key_col": "Code",
+    "key_is_code": True,
+    "rename_dict": {"People per Sq Km": "pop_density_persqkm"},
+    "keep_cols": ["LSOA11CD", "pop_density_persqkm"],
+    "csv_name": "pop_density_persqkm",
+}
+
+la_popdensity_dict = {
+    "data": raw.POPDENSITY_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 1",
+    "key_is_code": False,
+    "rename_dict": {"Mid-year 2018 ": "pop_density_persqkm"},
+    "keep_cols": [],
+    "csv_name": "pop_density_persqkm",
+}
+
+la_vulnerable_dict = {
+    "data": raw.VULNERABLE_LA,
+    "res": "LA",
+    "key_col": "index",
+    "key_is_code": False,
+    "rename_dict": {38: "vulnerable"},
+    "bracketed_data_cols": ["vulnerable"],
+    "keep_cols": [],
+    "csv_name": "vulnerable_count_percent",
+}
+
+la_cohesion_dict = {
+    "data": raw.COMM_COHESION_LA,
+    "res": "LA",
+    "key_col": "index",
+    "key_is_code": False,
+    "rename_dict": {20: "belong_strongagree", 21: "belong_agree"},
+    "bracketed_data_cols": ["belong_strongagree", "belong_agree"],
+    "keep_cols": [],
+    "csv_name": "comm_cohesion_count_percent",
+}
+
+la_internet_access_dict = {
+    "data": raw.INTERNET_ACCESS_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 0",
+    "key_is_code": False,
+    "rename_dict": {"Yes (%)": "has_internet_percent"},
+    "keep_cols": [],
+    "csv_name": "has_internet_percent",
+}
+
+la_internet_use_dict = {
+    "data": raw.INTERNET_USE_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 0",
+    "key_is_code": False,
+    "rename_dict": {
+        "Several times a day (%)": "use_several_daily_percent",
+        "Daily (%) ": "use_daily_percent",
+    },
+    "keep_cols": [],
+    "csv_name": "use_internet_percent",
+}
+
+la_ethnicity_dict = {
+    "data": raw.ETHNICITY_LA,
+    "res": "LA",
+    "key_col": "Unnamed: 1",
+    "key_is_code": False,
+    "keep_cols": [],
+    "csv_name": "ethnicities_percent",
+}
+
+print("Dictionaries loaded.")
+
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+# Apply functions to each dataset, and create new constant
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+WELSH_LSOA = clean_data(**lsoa_welsh_dict)
```

Comment:
in this case, you just also need to change the signature of `clean_data` no more expecting a dict as parameter but an instance of your data class :)